<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsPointDisplacementRendererWidgetBase</class>
 <widget class="QWidget" name="QgsPointDisplacementRendererWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="mRendererLabel">
     <property name="text">
      <string>Renderer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="mRendererComboBox"/>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QgsCollapsibleGroupBoxBasic" name="mLabelingGroupBox">
     <property name="title">
      <string>Labels</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="mLabelDistanceLabel">
        <property name="text">
         <string>Label distance factor</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mLabelColorLabel">
        <property name="text">
         <string>Label color</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QgsScaleWidget" name="mMinLabelScaleWidget">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mLabelAttributeLabel">
        <property name="text">
         <string>Label attribute</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="mScaleDependentLabelsCheckBox">
        <property name="text">
         <string>Use scale dependent labeling</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QgsColorButton" name="mLabelColorButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Label font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="mLabelFieldComboBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="mMaxScaleLabel">
        <property name="text">
         <string>Minimum map scale</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QgsFontButton" name="mLabelFontButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QgsDoubleSpinBox" name="mLabelDistanceFactorSpinBox">
        <property name="minimum">
         <double>-5.000000000000000</double>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Placement method</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="mDistanceToleranceLabel">
     <property name="toolTip">
      <string>Point distance tolerance</string>
     </property>
     <property name="text">
      <string>Distance</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="mRendererSettingsButton">
     <property name="text">
      <string>Renderer Settingsâ€¦</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QgsDoubleSpinBox" name="mDistanceSpinBox">
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="maximum">
        <double>999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsUnitSelectionWidget" name="mDistanceUnitWidget" native="true">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="mPlacementComboBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mCenterSymbolLabel">
     <property name="text">
      <string>Center symbol</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QgsCollapsibleGroupBoxBasic" name="mDisplacementCirclesGroupBox">
     <property name="title">
      <string>Displacement Lines</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QgsColorButton" name="mCircleColorButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mCircleRadiusLabel">
        <property name="text">
         <string>Size adjustment</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mCircleWidthLabel">
        <property name="text">
         <string>Stroke width</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mCircleColorLabel">
        <property name="text">
         <string>Stroke color</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QgsDoubleSpinBox" name="mCircleModificationSpinBox">
        <property name="suffix">
         <string> mm</string>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsDoubleSpinBox" name="mCircleWidthSpinBox">
        <property name="suffix">
         <string> mm</string>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsSymbolButton" name="mCenterSymbolToolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFontButton</class>
   <extends>QToolButton</extends>
   <header>qgsfontbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsScaleWidget</class>
   <extends>QWidget</extends>
   <header>qgsscalewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSymbolButton</class>
   <extends>QToolButton</extends>
   <header>qgssymbolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBoxBasic</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsUnitSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsunitselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mCenterSymbolToolButton</tabstop>
  <tabstop>mRendererComboBox</tabstop>
  <tabstop>mRendererSettingsButton</tabstop>
  <tabstop>mDistanceSpinBox</tabstop>
  <tabstop>mDistanceUnitWidget</tabstop>
  <tabstop>mPlacementComboBox</tabstop>
  <tabstop>mCircleWidthSpinBox</tabstop>
  <tabstop>mCircleColorButton</tabstop>
  <tabstop>mCircleModificationSpinBox</tabstop>
  <tabstop>mLabelFieldComboBox</tabstop>
  <tabstop>mLabelFontButton</tabstop>
  <tabstop>mLabelColorButton</tabstop>
  <tabstop>mScaleDependentLabelsCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
